app/views/govuk_publishing_components/components/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  add_gem_component_stylesheet("secondary-navigation")
+
+  id ||= nil
+  items ||= []
+%>
+<%= tag.nav class: "gem-c-secondary-navigation", id: id, role: "navigation", aria: { label: aria_label } do %>
+  <%= tag.ul class: "gem-c-secondary-navigation__list" do %>
+    <% items.each do |item| %>
+      <%
+        item_classes = %w( gem-c-secondary-navigation__list-item )
+        item_classes << "gem-c-secondary-navigation__list-item--current" if item[:current]
+        item_aria_attributes = { current: "page" } if item[:current]
+      %>
+      <%= tag.li class: item_classes do %>
+        <%= link_to item[:label], item[:href], class: "govuk-link govuk-link--no-visited-state gem-c-secondary-navigation__list-item-link", data: item[:data_attributes], aria: item_aria_attributes %>
+      <% end %>
+    <% end %>
+  <% end %>
+<% end %>
```

Comment:
Is it worth wrapping the render in an `if items` statement, so that there isn't a scenario where an empty nav/ul is rendered? (I appreciate this may be a very rare edge case)

Reply:
Yes, probably rare but worth it - have now add that.